application/app/components/fiches/shared/PotentielSolaireCard.tsx

```diff
@@ -0,0 +1,141 @@
+import { CircleHelp, HousePlug, University, Users, Zap } from 'lucide-react';
+
+const UNKNOWN_TEXTS = {
+	potentiel_solaire: '—',
+	nb_etablissements: '—',
+	nb_eleves: "Nombre d'élèves concernés inconnu",
+};
+
+const SOLAR_TEXT = {
+	high: 'Le potentiel solaire de cet établissement me paraît tout à fait rayonnant',
+	low: 'Des optimisations sont à prévoir pour un bon potentiel solaire',
+};
+
+const FOYER_CONSO_KWH = 2300;
+const PERSONNES_PAR_FOYER = 2;
+const HIGH_SOLAR_THRESHOLD = 500_000;
+
+type EtablissementNiveau = 'lycee' | 'college' | 'primaire' | 'etablissements';
+
+function getNiveauLabel(niveau?: EtablissementNiveau): string {
+	if (!niveau) return '-';
+	switch (niveau) {
+		case 'primaire':
+			return "d'écoles primaires";
+		case 'college':
+			return 'de collèges';
+		case 'lycee':
+			return 'de lycées';
+		case 'etablissements':
+			return "d'établissements";
+		default:
+			throw new Error(`Niveau inconnu : ${niveau}`);
+	}
+}
+
+interface PotentielSolaireCardProps {
+	potentiel_solaire?: number;
+	nb_etablissements?: number;
+	niveau?: EtablissementNiveau;
+	nb_eleves?: number;
+	showInterpretation?: boolean;
+	showNbEtablissements?: boolean;
+}
+
+function potentielSolaireEnFoyers(potentiel?: number): number | string {
+	if (potentiel === undefined) return UNKNOWN_TEXTS.potentiel_solaire;
+	return Math.round(potentiel / FOYER_CONSO_KWH / PERSONNES_PAR_FOYER);
+}
+
+function potentielSolaireEnMWh(potentiel?: number): number | string {
+	if (potentiel === undefined) return UNKNOWN_TEXTS.potentiel_solaire;
+	return Math.round(potentiel / 1000);
+}
```

Comment:
Tu peux reutiliser des fonctions pour convertir et formatter !
```suggestion
function getFormattedPotentielSolaire(potentiel?: number): number | string {
	if (potentiel === undefined) return UNKNOWN_TEXTS.potentiel_solaire;
	return formatNumber(convertKWhTo(potentiel, 'MWh')); 
}
```